R/read_habitatdata.R

```diff
@@ -797,8 +805,36 @@ read_habitatmap <-
 
         if (filter_hab) {
 
+            # version control: version habitatmap == version habitatmap_stdized
+            xxh64sum_habitatmap_stdized_present <- xxh64sum(file.path(
+                fileman_up("n2khab_data"),
+                "20_processed/habitatmap_stdized/habitatmap_stdized.gpkg"))
+
+            if (version == "habitatmap_2020") {
+                xxh64sum_habitatmap_stdized_expected <- "3109c26f0a27a0f3"
+            } else {
+                xxh64sum_habitatmap_stdized_expected <- c("b80f469f33636c8b","8e9c4e09f5f67c3e")
+            }
+
+            if (!(xxh64sum_habitatmap_stdized_present %in%
+                xxh64sum_habitatmap_stdized_expected)) {
+                stop("You are trying to use habitatmap version '", version,"' ",
+                     "with another version of habitatmap_stdized. ",
+                     "Specify the correct version as argument (version =) ",
+                     "and add the corresponding files under ",
+                     "'n2khab_data/10_raw/habitatmap' and ",
+                     "'n2khab_data/10_processed/habitatmap_stdized'.",
```

Comment:
Should use `20_processed`. Covered already in PR #140, where these lines were moved.